# Define the configuration message for robot position and image measurements
geometry_msgs/Point p1  # Camera position 1
geometry_msgs/Point d1 # Pixel coordinates from camera 1
float64 theta1  # Orientation angle from Y axis for camera 1
geometry_msgs/Point p2  # Camera position 2
geometry_msgs/Point d2 # Pixel coordinates from camera 2
float64 theta2  # Orientation angle from Y axis for camera 2